src/main/java/com/alipay/oceanbase/rpc/bolt/transport/ObTableRemoting.java

```diff
@@ -56,6 +57,12 @@ public ObPayload invokeSync(final ObTableConnection conn, final ObPayload reques
         request.setUniqueId(conn.getUniqueId());
 
         if (request instanceof Credentialable) {
+            if (conn.getCredential() == null) {
+                logger.warn("credential is null");
+                String errMessage = TraceUtil.formatTraceMessage(conn, request, "credential is null");
+                logger.warn(errMessage);
```

Comment:
the line 63 warning message is repetitive with line 61

Reply:
fixed